index.html

```diff
@@ -1,140 +1,170 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<title>Git for Windows</title>
-		<meta name="description" content="We bring the awesome Git VCS to Windows">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
-		<link rel="stylesheet" href="css/pack.css">
-		<link rel="shortcut icon" href="favicon.ico">
-		<!--[if lt IE 9]>
+
+  <head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Git for Windows</title>
+	<meta name="description" content="We bring the awesome Git VCS to Windows">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="robots" content="all,follow">
+	<!-- Bootstrap CSS-->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
+	<!-- Custom Stylesheet -->
+	<link rel="shortcut icon" href="favicon.ico">
+	<link rel="stylesheet" href="css/custom.css">
+	<!--[if lt IE 9]>
 		<script src="js/html5shiv.js"></script>
 		<script src="js/respond.js"></script>
 		<![endif]-->
-	</head>
-	<body>
-		<header>
-			<nav class="content">
-				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
-				<div class="version"><a href="https://github.com/git-for-windows/git/releases/tag/v2.34.0.windows.1" title="Version 2.34.0 was published on Mon, 15 Nov 2021 19:52:38 UTC">Version 2.34.0</a></div>
-				<ul class="list-unstyled">
-					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
-					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
-					<li><a href="https://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
-				</ul>
-			</nav>
-			<section class="content feature">
-				<div class="imagerollover">
-					<img src="img/gwindows_logo.png" alt="Git Logo">
-					<img class="top" src="img/git_logo.png" alt="Git Logo">
+  </head>
+
+  <body>
+	<!-- navbar -->
+	<nav class="navbar navbar-light navbar-expand-lg fixed-top shadow-sm bg-white">
+		<a href="index.html" class="navbar-brand">
+			<h1 class="gittext lowercase">
+				Git<span>for Windows</span>
+			</h1>
+		</a>
+		<button type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+			aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"
+			class="navbar-toggler"><span class="navbar-toggler-icon"></span></button>
+		<div id="navbarSupportedContent" class="collapse navbar-collapse">
+			<ul class="navbar-nav ml-auto">
+				<li class="nav-item"><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ"
```

Comment:
The "FAQ" link being bold and the others not, makes it stand out for the wrong reason.
Looks like "FAQ" is a category or label for the other links to the right, rather than being a link itself.

Reply:
FAQ is a link to Github's wiki page where FAQ content is. In the original design too, it was bold, so remaining true. We can tone down the boldness.

Reply:
Yes, let's remove the boldness. The original idea was to guide users to the FAQ, but I don't believe that it worked.

Reply:
I believe it's bold because it's an internal link, but the others are external. If you do want to un-bold it, you could use an icon indicator (like a [box arrow](https://icons.getbootstrap.com/icons/box-arrow-up-right/)) to show that the external links are external (and open in a new tab).

Reply:
IIRC I emboldened the link to provide an easy visual for those looking for the FAQs, and consequently the whole wiki, to find.  I'd hate to loose the 'easy to find' aspect.  Alternate designs that help folks find, and look up, all those FAQ & wiki items would be just as reasonable.

Reply:
I don't think that our strategy worked. The FAQ was not easy to find, neither was the wiki.

But maybe that's all okay, now that we have GitHub Discussions. It seems the StackOverflow-like format fits more people's needs than the wiki format.